turbinia/evidence.py

```diff
@@ -38,6 +41,65 @@
 log = logging.getLogger('turbinia')
 
 
+def evidence_class_names(all_classes=False):
+  """Returns a list of class names for the evidence module.
+
+  Args:
+    all_classes (bool): Flag to determine whether to include all classes
+        in the module.
+  """
+  predicate = lambda member: inspect.isclass(member) and not inspect.isbuiltin(
+      member)
+  class_names = inspect.getmembers(sys.modules[__name__], predicate)
+  if not all_classes:
+    # TODO: Non-evidence types should be moved out of the evidence module,
+    # so that we no longer have to ignore certain classes here. Especially
+    # 'output' and 'report' types.
+    # Ignore classes that are not real Evidence types and the base class.
+    ignored_classes = (
+        'BinaryExtraction', 'BodyFile', 'BulkExtractorOutput', 'Evidence',
```

Comment:
I think we might not need to ignore `BodyFile` here as we do have jobs/tasks that can process it (e.g. Plaso).

Reply:
Just for clarity, classes in the ignore list won't mean they can't be processed. It only means that the new cli tool won't generate a command/cli args for it. Turbiniactl currently doesn't support BodyFile as an evidence type (i.e. you can't create new requests using turbiniactl to process bodyfile evidences).

In summary, removing it from this list means the new cli tool will allow a user to create new requests for bodyfile evidence types. Sounds good to me, just wanted to make sure this is what we intend to do here.

Reply:
Done